extensions/sharing/pkg/config/config.go

```diff
@@ -48,69 +55,70 @@ type Debug struct {
 }
 
 type GRPCConfig struct {
-	Addr     string `yaml:"addr" env:"SHARING_GRPC_ADDR" desc:"The address of the grpc service."`
-	Protocol string `yaml:"protocol" env:"SHARING_GRPC_PROTOCOL" desc:"The transport protocol of the grpc service."`
+	Addr      string `yaml:"addr" env:"SHARING_GRPC_ADDR" desc:"The address of the grpc service."`
+	Namespace string `yaml:"-"`
+	Protocol  string `yaml:"protocol" env:"SHARING_GRPC_PROTOCOL" desc:"The transport protocol of the grpc service."`
 }
 
 type UserSharingDrivers struct {
-	JSON UserSharingJSONDriver
-	SQL  UserSharingSQLDriver
-	CS3  UserSharingCS3Driver
+	JSON UserSharingJSONDriver `yaml:"json"`
+	CS3  UserSharingCS3Driver  `yaml:"cs3"`
+	SQL  UserSharingSQLDriver  `yaml:"sql,omitempty"` // not supported by the oCIS product, therefore not part of docs
```

Comment:
add `owncloudsql`?

Reply:
I think `oc10-sql` got renamed to `owncloudsql` as well, so the `sharing/pkg/revaconfig/config.go` might need an update as well